AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -254,6 +254,12 @@ extern NSInteger const ASDefaultDrawingPriority;
 
 /** @name Managing dimensions */
 
+// TODO: coalesc: Documentation
+- (void)invalidateSize;
+- (void)didInvalidateSize;
+- (void)sizeToFit;
+- (CGSize)sizeThatFits:(CGSize)size;
```

Comment:
Probably good to move sizeToFit and sizeThatFits into the +FrameworkPrivate.h header as we're experimenting with them - I don't think they are yet crucial external API.

The functionality of -invalidateSize is definitely crucial external API in the first version.  We can think of a final name later once all the other code is done & landed.